website/content/partials/telemetry-metrics/vault/raft/snapshot/persist.mdx

```diff
@@ -0,0 +1,5 @@
+### vault.raft.snapshot.persist ((#vault-raft-snapshot-persist))
+
+Metric type | Value | Description
+----------- | ----- | -----------
+timer       | ms    | Time required to write the current snapshot to disk
```

Comment:
This isn't quite correct, because when taking a snapshot, we never write it to disk, we just stream it out to the client.  We do persist a tiny value to disk here, so maybe: "Time spent recording snapshot meta information to disk when taking snapshots".

Reply:
sgtm. updated